src/main/resources/tags/help/portforward.tag

```diff
@@ -1,10 +1,12 @@
 type: text
+title: :tools: Test your Server's Port Forwarding
+color: neutral
 image: https://i.ibb.co/7J0HhZF/Portforward.png
+button: [IPFingerPrints](https://www.ipfingerprints.com/portscan.php)
 
 ---
 
-**Test your server's port forwarding**
-1. Go [here](https://www.ipfingerprints.com/portscan.php)
-2. Follow the guide on this picture. Use the port that your Geyser uses (19132 is the default)
+1. Use the link below
```

Comment:
I don't think this needs to be a list anymore, just reword this to be "Using the Port Scanner tool accessible below, follow the guide..."